--- F03-before
+++ F03-after
@@ -2955,8 +2955,8 @@
         "2": "GND",
         "3": "TEST_KEY",
         "4": "MECH_OK",
-        "5": "LOGGER_CLEAR",
-        "6": "TEST_PRESENT",
+        "5": "NC",
+        "6": "NC",
         "7": "STOP_NC_OUT",
         "8": "NC",
         "9": "ARM_CONTACT"
@@ -7510,8 +7510,8 @@
         "2": "GND",
         "3": "TEST_KEY",
         "4": "MECH_OK",
-        "5": "LOGGER_CLEAR",
-        "6": "TEST_PRESENT",
+        "5": "NC",
+        "6": "NC",
         "7": "STOP_NC_OUT",
         "8": "NC",
         "9": "ARM_CONTACT"
@@ -9719,7 +9719,7 @@
       "from_pin": 5,
       "from_ref": "J_PANELSAFEA",
       "max_cm": 30.0,
-      "net": "LOGGER_CLEAR",
+      "net": "NC",
       "to_board": "P11",
       "to_pin": 5,
       "to_ref": "J_PANELSAFEB"
@@ -9731,7 +9731,7 @@
       "from_pin": 6,
       "from_ref": "J_PANELSAFEA",
       "max_cm": 30.0,
-      "net": "TEST_PRESENT",
+      "net": "NC",
       "to_board": "P11",
       "to_pin": 6,
       "to_ref": "J_PANELSAFEB"
