--- F07-before
+++ F07-after
@@ -2252,7 +2252,7 @@
         "3": "TEST_KEY",
         "4": "LOGGER_CLEAR",
         "5": "TEST_PRESENT",
-        "6": "STATUS_LED",
+        "6": "NC",
         "7": "N_J_SCOPE_HOT",
         "8": "GND"
       },
@@ -7493,7 +7493,7 @@
         "3": "TEST_KEY",
         "4": "LOGGER_CLEAR",
         "5": "TEST_PRESENT",
-        "6": "STATUS_LED",
+        "6": "NC",
         "7": "N_J_SCOPE_HOT",
         "8": "GND"
       },
@@ -9851,7 +9851,7 @@
       "from_pin": 6,
       "from_ref": "J_PANELCOREA",
       "max_cm": 30.0,
-      "net": "STATUS_LED",
+      "net": "NC",
       "to_board": "P11",
       "to_pin": 6,
       "to_ref": "J_PANELCOREB"
